.github/workflows/release.yml

```
@@ -37,7 +37,7 @@ jobs:
       - name: Checkout master
         run: git checkout master && git pull --tags
       - name: Bump Version
-        run: npm version ${{ github.event.inputs.type }} --message "[RELEASE] %s"
+        run: npm run version-prepare && npm version ${{ github.event.inputs.type }} --message "[RELEASE] %s"
```

No need to change the command if we rename **_version-prepare_** in **_preversion_** in the script of package.json.

```suggestion
        run: npm version ${{ github.event.inputs.type }} --message "[RELEASE] %s"
```